Makefile.devcontainer

```diff
@@ -75,6 +75,7 @@ FILES = $(shell find . -type f -name '*.go')
 
 format:                         ## Format source code.
 	gofumpt -l -w $(FILES)
+	gci write --Section Standard --Section Default --Section "Prefix(github.com/percona/pmm-managed)" .
 	goimports -local github.com/percona/pmm-managed -l -w $(FILES)
```

Comment:
Shouldn't we run `gci` after `goimports`? Not sure but I imagined gci will only order them and it makes sense to do that after goimports have fixed missing/unused imports or something.

```suggestion
        goimports -local github.com/percona/pmm-managed -l -w $(FILES)
	gci write --Section Standard --Section Default --Section "Prefix(github.com/percona/pmm-managed)" .
```

Reply:
good point